pcdsdevices/smarpod.py

```diff
@@ -0,0 +1,133 @@
+    ver_product = Cpt(EpicsSignalRO, ':VER:PRODUCT', kind='omitted')
+    ver_firmware = Cpt(EpicsSignalRO, ':VER:FIRMWARE', kind='omitted')
+    # Status
+    cmd_read_status = Cpt(EpicsSignal, ':CMD:READ_STATUS', kind='normal')
+    status_temp = Cpt(EpicsSignalRO, ':STATUS:TEMP', kind='omitted')
+    status_load = Cpt(EpicsSignalRO, ':STATUS:LOAD', kind='omitted')
+    status_memory = Cpt(EpicsSignalRO, ':STATUS:MEMORY', kind='omitted')
+    status_net_bytes_in = Cpt(EpicsSignalRO, ':STATUS:NET_BYTES_IN', kind='omitted')
+    status_net_bytes_out = Cpt(EpicsSignalRO, ':STATUS:NET_BYTES_OUT', kind='omitted')
+    status_net = Cpt(EpicsSignalRO, ':STATUS:NET', kind='omitted')
+    status_status = Cpt(EpicsSignalRO, ':STATUS:STATUS', kind='omitted')
+    status_bootcnt = Cpt(EpicsSignalRO, ':STATUS:BOOTCNT', kind='omitted')
+    status_uptime = Cpt(EpicsSignalRO, ':STATUS:UPTIME', kind='omitted')
+    status_ip = Cpt(EpicsSignalRO, ':STATUS:IP', kind='omitted')
+    status_client_ip = Cpt(EpicsSignalRO, ':STATUS:CLIENT_IP', kind='omitted')
+    status_ser_status = Cpt(EpicsSignalRO, ':STATUS:SER_STATUS', kind='omitted')
+    status_ser_bytes_in = Cpt(EpicsSignalRO, ':STATUS:SER_BYTES_IN', kind='omitted')
+    status_ser_bytes_out = Cpt(EpicsSignalRO, ':STATUS:SER_BYTES_OUT', kind='omitted')
+    status_display = Cpt(EpicsSignalRO, ':STATUS:DISPLAY', kind='omitted')
+    # Sensor mode
+    cmd_sensor_mode = Cpt(EpicsSignal, ':CMD:SENSOR_MODE', kind='normal')
+    sensor_mode = Cpt(EpicsSignal, ':SENSOR_MODE', kind='normal', doc='Sensor mode')
+    # Maximum frequency
+    cmd_freq = Cpt(EpicsSignal, ':CMD:FREQ', kind='normal')
+    freq = Cpt(EpicsSignal, ':FREQ', kind='normal')
+    # Maximum velocity
+    cmd_vel = Cpt(EpicsSignal, ':CMD:VEL', kind='normal', doc='sets movement velocity')
+    vel = Cpt(EpicsSignal, ':VEL', kind='normal', doc='current speed-control and speed settings')
+    # Maximum acceleration
+    cmd_accel = Cpt(EpicsSignal, ':CMD:ACCEL', kind='normal', doc='sets movement acceleration')
+    accel = Cpt(EpicsSignal, ':ACCEL', kind='normal', doc='current acceleration-control and acceleration settings')
+# Read and Set Poses, Pose 1-5:
+
+
+class SmarpodPose(Device):
+    pose_name = Cpt(EpicsSignal, ':NAME_RBV', write_pv=':NAME', kind='normal', doc='Pose name input')
+    x = Cpt(EpicsSignal, ':X', kind='normal', doc='Pose X Position Data')
+    y = Cpt(EpicsSignal, ':Y', kind='normal', doc='Pose Y Position Data')
+    z = Cpt(EpicsSignal, ':Z', kind='normal', doc='Pose Z Position Data')
+    rx = Cpt(EpicsSignal, ':RX', kind='normal', doc='Pose RX Position Data')
+    ry = Cpt(EpicsSignal, ':RY', kind='normal', doc='Pose RY Position Data')
+    rz = Cpt(EpicsSignal, ':RZ', kind='normal', doc='Pose RZ Position Data')
+    record = Cpt(EpicsSignal, ':RECORD', kind='normal', doc='Fanout for pose')
+    execute = Cpt(EpicsSignal, ':EXECUTE', kind='normal', doc='Enacts Pose Positions')
+
+
+class Smarpod(Device):
```

Comment:
There's `SmarPod` above and `Smarpod` here - two different classes.
The above one has everything but the poses. I think these should be merged somehow?